### PR TITLE
Changes based on CR (use devicegraph and Devicegraph to be consistent)

### PR DESCRIPTION
**Pull request by teclator**
To be discussed:

Although by convention we should use `camel case` 'Device Graph' class and snake case for `device_graph` variable or methods, it is already not the case in Y2Storage, and they decided so.

If we use different conventions then it will be error prone.

This PR not only rename the `Y2Partitioner::DeviceGraphs` singleton class to `Y2Partitioner::Devicegraph` but also all the references to the already commented variables and methods.

### CODE REVIEW

File: src/lib/y2partitioner/devicegraph.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+module Y2Partitioner
+  # Singleton class that helps to work with a copy of the system
+  # {Y2Storage::Devicegraph}
+  class Devicegraph
```
Comment (ancorgs) [inline src/lib/y2partitioner/devicegraph.rb]: Shouldn't this be `Devicegraphs` in plural? An instance of the class does not represent a single devicegraph, but a singleton object containing several devicegraphs. I'm not a fan of class names in plural, but I like even less class names that don't match expectations.

Comment (teclator) [inline src/lib/y2partitioner/devicegraph.rb]: @ancorgs true, it is even worst. Renamed it back to origin.

**Comment by ancorgs** [comment]
After the latest changes, I like this pull request. I think it makes the name more consistent with libstorage and yast-storage and I prefer the new API in which `current` is "dupped" from the initial devicegraph during initialization.

But @mvidner should take a look, as father of the creature.

**Comment by mvidner** [comment]
Sorry, I, as the High Priest of English Grammar, and a Knight of Spelling,  refuse to make the spelling wrong. Device and Graph are simply two words. (Unlike [Filename](https://en.wikipedia.org/wiki/Filename) where the usage varies). Consistency is also important so I would agree to making `Devicegraph` an alias to `DeviceGraph`.

The `create_instance` part looks good, thanks!

**Comment by ancorgs** [comment]
Then we will have a `DeviceGraphs` object used to store some important `Devicegraph` objects. Fine with me, but be prepared for some IToldYous when we make a mistake. :wink:

**Comment by teclator** [comment]
Closed this PR as some of the changes have been already added to the #5 one.

**Pull request closed without merging**